### PR TITLE
See more is inside the card

### LINKED ISSUE #2679: [UI] See more is out of card
**Description**

In the "Service Mesh Specifications and Why They Matter" card the **see more** is out of card on screen size 990 x 1080 

**Expected Behavior**

The **see more**  should be in under the card.

**Screenshots**

Screenshot from 2022-03-23 13-12-35

**Environment:**
- Host OS: Linux
- Browser: chrome

---

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Akshit42-hue**
**Description**

This PR fixes #2679 

**Notes for Reviewers**
Now the see more is inside the card

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Akshit42-hue** [comment]
@Nikhil-Ladha please do have a look

**Comment by Akshit42-hue** [comment]
@Abhijay007 @Nikhil-Ladha can you please give me any update

File: src/components/Card/Card.style.js [hunk 1]
```diff
@@ -50,15 +50,16 @@ export const CardWrapper = styled.div`
     }
 
     .post-content-block{
-        padding: 1rem 2rem;
+        padding: 0rem 2rem;
```
Comment (Nikhil-Ladha) [inline src/components/Card/Card.style.js:53]: Make the change in the blog CSS only, and not in the component CSS.

**Comment by Akshit42-hue** [comment]
@Nikhil-Ladha Please do have a look

**Comment by l5io** [comment]
🚀 Preview for commit 0d5be154d7ec60bc12fbb98c7e4d64d667050d5d at: https://623ea2255b53867cfc136a1f--layer5ng.netlify.app

**Review by Abhijay007 — APPROVED** [review]
LGTM !! 👍

**Comment by Akshit42-hue** [comment]
@Abhijay007 @Nikhil-Ladha can you please add **openforce** label

**Comment by Abhijay007** [comment]
> @Abhijay007 @Nikhil-Ladha can you please add **openforce** label

done :)

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
You are still changing the CSS of the component. Please revert that.
And, only update the CSS for the Card component in the blogs code itself.

**Comment by Akshit42-hue** [comment]
I am sorry but I am not understanding what you are trying to say.Can you please explain more about it @Nikhil-Ladha 
 Only the "Service Mesh Specifications and Why They Matter" card css I want to change. Am I right

**Comment by Nikhil-Ladha** [comment]
Make the change to the [Blog-grid CSS](https://github.com/layer5io/layer5/blob/master/src/sections/Blog/Blog-grid/blogGrid.style.js) only. So, that other pages which are using the Card component don't get affected by the fixed height change.

**Comment by Akshit42-hue** [comment]
Thanks for clarifying @Nikhil-Ladha

**Comment by Akshit42-hue** [comment]
Please do have a look @Nikhil-Ladha

**Comment by l5io** [comment]
🚀 Preview for commit 9e8d7c12fad06073333f3a5499af057180d632f0 at: https://62459d43f065fd520cd4a97b--layer5ng.netlify.app

File: src/sections/Blog/Blog-grid/blogGrid.style.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ export const BlogPageWrapper = styled.div`
     .blog-grid-wrapper{
         margin: 5rem auto;
     }
+    .post-meta-block{
+        height: 3.5rem;
+
```
Comment (Nikhil-Ladha) [inline src/sections/Blog/Blog-grid/blogGrid.style.js:9]: Move this under the below `media` tag for `max-width: 992px`, above that it is not required.

**Comment by Akshit42-hue** [comment]
Required changes have been done @Nikhil-Ladha

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**